src/lib/gui/config/AppConfig.cpp

```diff
@@ -471,12 +463,20 @@ QString AppConfig::lastVersion() const
 
 QString AppConfig::coreServerName() const
 {
-  return m_CoreServerName;
+#ifdef Q_OS_WIN
+  return s_2argTemplate.arg(s_CoreServerName, s_exeExt);
```

Comment:
This is more readable.
```suggestion
  return QStringLiteral("%1.exe").arg(s_CoreServerName);
```